src/ui/components/Card/index.js

```diff
@@ -53,28 +63,21 @@ export default class Card extends React.Component {
           'Card--no-header': !header,
           'Card--no-footer': !footer,
         })}
-        ref={(ref) => { this.cardContainer = ref; }}
```

Comment:
We don't need these `ref`s anymore?

Reply:
Ah, it looks like we only needed them for the tests. Is that correct?

Reply:
Yes, we rarely use `ref` in source files.